bftengine/src/bftengine/ReplicaLoader.cpp

```diff
@@ -48,14 +48,22 @@ namespace {
 ReplicaLoader::ErrorCode loadConfig(LoadedReplicaData &ld) {
   ld.repsInfo = new ReplicasInfo(ld.repConfig, dynamicCollectorForPartialProofs, dynamicCollectorForExecutionProofs);
   auto &config = ld.repConfig;
-
-  ld.sigManager = SigManager::init(config.replicaId,
-                                   config.replicaPrivateKey,
-                                   config.publicKeysOfReplicas,
-                                   concord::crypto::KeyFormat::HexaDecimalStrippedFormat,
-                                   ReplicaConfig::instance().getPublicKeysOfClients(),
-                                   concord::crypto::KeyFormat::PemFormat,
-                                   *ld.repsInfo);
+  std::string operator_key;
+  std::ifstream op_key_file(bftEngine::ReplicaConfig::instance().pathToOperatorPublicKey_);
```

Comment:
since this code is duplicated, add a sigmanager ctor which receives the path, parses the key and adds an operator. The operator should be an optional parameter, maybe even with std::optional.

Reply:
done